.changeset/gentle-monkeys-flash.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux/adp-tooling': minor
```

Comment:
Just make it a patch please. I would consider all the changes fixes and not new features.